### PR TITLE
Improve user UI experience

### PR DESCRIPTION
**Pull request by LouiaR**
Improve user’s UI experience:

    1. Improve design of  Organisation card 
    • Make padding, marging , font-size, width, police consistent 
    • Add icons
    • Display on each card organisation’ name, their location (area, borough), process and  services it offers.
    2. Improve design of Edit organisation form
    •   Display label for every field 
    •    Re-order input fields
    •    Make padding, marging , font-size, width, police consistent
    •    Add  tag icon
    3. Improve design of  Add organisation form 
WIP

### CODE REVIEW

File: backend/src/routes/users.js [hunk 1]
```diff
@@ -18,7 +18,14 @@ const router = express.Router();
 
 router.get('/users', async (req, res) => {
   try {
-    await getAllUsers().then(users => res.status(200).json(users));
+    const usersArr = []
+    await getAllUsers().then(users => users.map(user => usersArr.push({
```
Comment (kkarimi) [inline backend/src/routes/users.js]:
```
 await getAllUsers().then(users => users.map(user => usersArr.push({
```
becomes:
```
const users =  await getAllUsers()
users.map(user => usersArr.push({...})
```

File: frontend/src/actions/getApiData.js [hunk 2]
```diff
@@ -74,3 +74,17 @@ export function getOrganisationsList(){
...
+export function setListOfUsers(users) {
+  return{
+    type:LIST_OF_USERS,
+    users
+  }
+}
+
+export function getListOfUsers(){
+  return dispatch => {
+    axios.get(`${api}/users`)
```
Comment (kkarimi) [inline frontend/src/actions/getApiData.js]: Please use try/catch

File: backend/src/routes/users.js (hunk 1 above)
Comment (LouiaR) [reply backend/src/routes/users.js]: Good spot @kkarimi I solve it on PR 119 ,  I accidentally pop a stash when working on this PR. I think by merging the recent changes I can get rid of this or do you have have another way to remove this commits

File: frontend/src/actions/getApiData.js (hunk 2 above)
Comment (LouiaR) [reply frontend/src/actions/getApiData.js]: Ditto

**Comment by LouiaR** [comment]
It contains unwanted commits

**Pull request closed without merging**